apps/design-land/src/app/core/code-preview/component/code-preview.component.html

```diff
@@ -0,0 +1,10 @@
+<daff-navbar color="primary" *ngIf="title">
```

Comment:
The navbar doesn't get rendered on the page.

Reply:
It does get rendered it's just not in use because none of the examples are using the `title` input property and instead are using a separate `h2` header. The examples could be redone to all use this title header instead or it could be removed.

Reply:
I wouldn't use the `<daff-navbar>` component to place the title, but you can leave it for now. I'll change it when I do the style updates.